playwright.config.js

```diff
@@ -5,8 +5,14 @@ const baseURL = "http://127.0.0.1:8080";
 export default defineConfig({
```

Comment:
Copied from https://github.com/Qiskit/qiskit_sphinx_theme/pull/376. I generated the snapshot built on top of that PR, so am copying over the change here so the snapshot works.